examples/working-directly-with-json/working_directly_with_json.md

```diff
@@ -8,4 +8,6 @@ allowed on lax types. For such operations, some of the type checking is moved fr
 
 ::: code working_directly_with_json.bal :::
 
+Run the example as follows.
```

Comment:
Not related to this BBE?

Reply:
It is better to give a general instruction to run it in every BBE as the `bal run` applies to all BBEs.

Reply:
I meant that this was the Hello World BBE PR. The change seems to be there in https://github.com/ballerina-platform/ballerina-distribution/pull/2927 (https://github.com/ballerina-platform/ballerina-distribution/pull/2927/files#diff-b5e63065ea56eb1e67041e03bdf6cadd64ab3f9d724a4b7da3234effb4f7228cR7) anyway.

Reply:
Yeah, it is better to add that line in all BBEs.

Reply:
Removed the MD file of `Working directly with JSON`, which was added by mistake.